lib/system.nim

```diff
@@ -1979,7 +1979,7 @@ proc compileOption*(option, arg: string): bool {.
 const
   hasThreadSupport = compileOption("threads") and not defined(nimscript)
   hasSharedHeap = defined(boehmgc) or defined(gogc) # don't share heaps; every thread has its own
-  taintMode = compileOption("taintmode")
+  taintMode* = compileOption("taintmode")
```

Comment:
Please don't export it and instead use ``compileOption("taintmode")`` elsewhere.

Reply:
Thanks for the comment! I've reverted this and defined `taintMode` separately in `streams.nim`.